packages/telegram-stamper/src/index.ts

```diff
@@ -0,0 +1,186 @@
+/// <reference lib="dom" />
+import { ApiKeyStamper } from "@turnkey/api-key-stamper";
+import { TelegramStamperError } from "./errors";
+
+declare global {
+  interface Window {
+    Telegram: any;
+  }
+}
+
+// config when telegram stamper is passed in an api key
+export type TTelegramStamperConfig = {
+  apiPublicKey: string;
+  apiPrivateKey: string;
+  cloudStorageKeySuffix?: string;
+};
+
+// Constants for default key names, note these should NOT be used directly, use the getPublicKeyCloudStorageKeyName() functions as they include the user passed key suffix
+const PUBLIC_KEY_CLOUD_STORAGE_KEY = "TURNKEY_API_PUBLIC_KEY";
+const PRIVATE_KEY_CLOUD_STORAGE_KEY = "TURNKEY_API_PRIVATE_KEY";
```

Comment:
Might be worth considering a single key that is a JSON value? That way there's no doubt about the public key and the private key matching. `setItem` can either succeed or fail which is good. If we have this split in two entries we need to worry about 2 setItem calls, and rollbacks and inconsistencies.

Reply:
I changed this so that the priv and public key are stored as a json object in 1 key in cloud storage. Theres a good bit of parsing, error, and type checking logic added as a result of this